go.mod

```diff
@@ -6,7 +6,7 @@ require (
 	github.com/google/go-cmp v0.6.0
 	github.com/hashicorp/go-version v1.6.0
 	github.com/hashicorp/hc-install v0.6.2
-	github.com/hashicorp/terraform-json v0.18.0
+	github.com/hashicorp/terraform-json v0.18.1-0.20231207151239-da9b6fa7714c
```

Comment:
This is temporary whilst [Add useJSONNumber to Plan and amend UnmarshalJSON behavior](https://github.com/hashicorp/terraform-json/pull/113#top) is being considered.

Reply:
Bumped to `terraform-json` `v0.19.0`